huggingface/pytorch/buildspec.yml

```diff
@@ -30,21 +41,39 @@ images:
     transformers_version: &TRANSFORMERS_VERSION 4.6.1
     datasets_version: &DATASETS_VERSION 1.6.2
     tag: !join [ *VERSION, '-', 'transformers', *TRANSFORMERS_VERSION, '-', *DEVICE_TYPE, '-', *TAG_PYTHON_VERSION, '-',
-                 *CUDA_VERSION, '-', *OS_VERSION ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /,
-                         *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      *CUDA_VERSION, '-', *OS_VERSION ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, 
+      *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+  BuildHuggingFacePytorchCPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &HUGGINGFACE_PYTORCH_CPU_INFERENCE_PY3 false
+    image_size_baseline: 4899
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py36
+    os_version: &OS_VERSION ubuntu18.04
+    transformers_version: &TRANSFORMERS_VERSION 4.6.0
```

Comment:
Yes, it should be `4.6.1` i updated it. We don't need `datasets` for inference. adding the `HF_INFERENCE_TOOLKIT_VERSION` here was only a suggestion/question if it would make sense to control a version like this or should we rather pin it inside the `dockerfile`? 
What do you suggest? controlling it with the `buildspec.yaml` or controlling it inside the `dockerfile`?